dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1108,260 @@ async def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=await call.initial_metadata())
 
+    async def start_workflow(
+            self,
+            workflow_component: str,
+            workflow_name: str,
+            input: any,
+            instance_id: Optional[str] = "",
```

Comment:
The default value should be `None` when using `Optional[str]`.

Reply:
Our API inside dapr/dapr checks against an empty string, shouldn't we leave it as "" rather than None?
Furthermore, the linter isn't the biggest fan of the change and gives the following error:
"Incompatible default for argument "instance_id" (default has type "None", argument has type "str")"